sci-libs/minc-toolkit/minc-toolkit-9999.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+inherit cmake-utils git-r3
+
+DESCRIPTION="Metaproject uniting all the available MINC tools"
+HOMEPAGE="https://github.com/BIC-MNI/minc-toolkit"
+SRC_URI=""
+EGIT_REPO_URI="git://github.com/BIC-MNI/minc-toolkit.git"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+KEYWORDS=""
+
+COMMON_DEP="dev-libs/libpcre
+	sci-libs/gsl
+	sci-libs/hdf5
+	sci-libs/itk
+	media-libs/freeglut
```

Comment:
Only required for `MT_BUILD_VISUAL_TOOLS`, thus use conditional dep. See https://github.com/BIC-MNI/minc-toolkit/blob/master/CMakeLists.txt#L376 and please be more accurate in case of dependencies.